frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwareTitleDetailsPage.tsx

```diff
@@ -110,12 +111,13 @@ const SoftwareTitleDetailsPage = ({
       refetchSoftwareTitle();
       return;
     }
+
+    const queryParams: string = buildQueryStringFromParams({
+      team_id: teamIdForApi,
+    });
+
     // redirect to software titles page if no versions are available
-    if (teamIdForApi) {
```

Comment:
changing this condition to `if (teamIdForApi !== API_ALL_TEAMS_ID)` or `(teamIdForApi !== undefined)` would also do the trick, and prevent the dangling `"?"` when no team_id is present

Reply:
Ooo, yeah, +1 to get rid of the dangling `?`. I just borrowed a solution pattern we had elsewhere in the code. We should definitely agree on the best pattern and clean up all these routes!